exampleCourse/elements/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {};
```

Comment:
Can we make this JSON rather than code, or get rid of it entirely? I'm thinking it would be nice if you could somehow just copy an element between courses or into core and it just gets picked up and works.

I'm just spitballing here, but what if each element had it's own directory, with an `infoElement.json` file or something in it, which would specify the main file (`pl_multiple_choice.py`) and client-side dependencies.

That way you someone could just copy that directory and drop it into their course element directory.

Thoughts? Problems with this?

Reply:
Putting each element in a directory was already starting to look like a good idea, as we start to add files (mustache, css, js, etc.) - so I'm in favor of that as well, I think.

Reply:
I do like the idea of using directory names as element names, that seems to fit well with our conventions. If we cache the element list in memory so we aren't reading the directory structure for each phase of each element, I think that'd work well!